### ISSUE TITLE
MinCount Sketch vs CountMin Sketch

### ISSUE THREAD

**Issue by r-gaia-cs** [ged-lab/khmer#779]
`doc/introduction.txt` has

~~~
(also see `MinCount Sketch
<http://www.eecs.harvard.edu/~michaelm/CS222/countmin.pdf>`__), which is
~~~

and `doc/user/choosing-table-sizes.txt` has

~~~
efficient way by taking advantage of two data structures, `Bloom
filters <http://en.wikipedia.org/wiki/Bloom_filter>`__ and `CountMin
Sketches <https://sites.google.com/site/countminsketch/>`__, that are
~~~

For a **novice** will be good to use the same notation and links if "MinCount Sketch" == "CountMin Sketch" and, otherwise, make the difference clear.

**Comment by mr-c** [comment]
Agreed, thanks.

**Comment by bocajnotnef** [comment]
Fixed by #782
